install_dependencies.sh

```diff
@@ -86,7 +85,7 @@ echo "Installing Udev rules for phidgets"
 echo "================================================================"
 
 # Setup udev rules
-sudo cp /usr/share/ros/phidgets_api/share/udev/99-phidgets.rules /etc/udev/rules.d
+sudo cp extended_pkg/phidgets_api/share/udev/99-phidgets.rules /etc/udev/rules.d
 echo "Phidgets udev rules have been copied to /etc/udev/rules.d"
 # The reason the script isn't used is because stupid bash/sh doesn't know which directory it's in
 # The actual script that comes with phidget is under /usr/share/ros/phidgets_api/share/setup-udev.sh
```

Comment:
This could be rephrased a bit to make a couple things more clear (and for conventions sake, moved to above the actual `cp`). Something like:
```
# While the `phidgets_api` does come with a script to do this setup for us,
# it must be run from within the folder it is contained in, and so will break if
# called by this script. As such, we just manually copy these rules.
```
I'm not mentioning the path to the the `setup-udev.sh` file, because it's likely to become out of date (in fact, it already is).